extensions.json

```
@@ -730,6 +730,12 @@
       "version": "10.2.0",
       "checkout": "v10.2.0"
     },
+    {
+      "id": "vscode.smali",
+      "repository": "https://github.com/ViRb3/vscode-smali",
+      "version": "1.0.0",
+      "location": "smali"
```

Here the default behavior of of the publishing script may be a bit weird: it always runs `npm install` or `yarn install` in the repo root, and not in the extension location.

To work around this weirdness, I suggest we "also" run `npm install` in the correct location using a `"prepublish"` command:

```suggestion
      "location": "smali",
      "prepublish": "cd smali && npm install"
```